src/main/java/org/example/cosmocatsintergalacticmarketplace/domain/Product.java

```diff
@@ -0,0 +1,21 @@
+package org.example.cosmocatsintergalacticmarketplace.domain;
+
+import lombok.*;
+
+/**
+ * DDD model: Product — reprsent goods on marketplace.
+ */
+@Data
+@Builder
```

Comment:
u can stick to one, builder or the constructor https://stackoverflow.com/questions/29881135/difference-between-builder-pattern-and-constructor in the end of the day, they both just different methods to do the same thing